src/rust/generators/sysmon-subgraph-generator/src/generator.rs

```diff
@@ -1,15 +1,46 @@
 use crate::metrics::SysmonSubgraphGeneratorMetrics;
 use crate::models::SysmonTryFrom;
 use async_trait::async_trait;
-use failure::bail;
+
 use grapl_graph_descriptions::graph_description::*;
 use grapl_observe::log_time;
 use log::*;
-use sqs_lambda::cache::{Cache, CacheResponse};
-use sqs_lambda::event_handler::{Completion, EventHandler, OutputEvent};
+use sqs_executor::cache::{Cache, CacheResponse};
+use sqs_executor::errors::{CheckedError, Recoverable};
+use sqs_executor::event_handler::{CompletedEvents, EventHandler};
+use sqs_executor::event_status::EventStatus;
 use std::borrow::Cow;
 use sysmon::Event;
 
+#[derive(thiserror::Error, Debug)]
+pub enum SysmonGeneratorError {
+    #[error("DeserializeError")]
+    DeserializeError(failure::Error),
+    #[error("ChronoError")]
+    NegativeEventTime(i64),
+    #[error("TimeError")]
+    TimeError(#[from] chrono::ParseError),
+    #[error("DeserializeError")]
+    GraphBuilderError(String),
+    #[error("Unsupported event type")]
+    UnsupportedEventType(String),
+    #[error("Generator failed")]
+    Unexpected,
```

Comment:
Many of these macros don't match their enum values (but in particular, NegativeEventTime and GraphBuilderError)